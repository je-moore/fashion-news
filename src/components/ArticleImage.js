import React from 'react'
import { useArticleImageStyles, DEFAULT_IMAGE } from '../styles'

function ArticleImage({ imageUrl }) {
  const classes = useArticleImageStyles()
  const articleImg = imageUrl
    ? classes.articleImg
    : classes.articleImg + ' ' + classes.defaultImg

  return (
    <img
      className={articleImg}
      src={imageUrl ? `https://fashionunited.com${imageUrl}` : DEFAULT_IMAGE}
      alt=""
    />
  )
}

export default ArticleImage
