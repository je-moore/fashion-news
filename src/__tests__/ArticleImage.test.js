import React from 'react'
import EnzymeToJson from 'enzyme-to-json'
import { mount } from 'enzyme'
import ArticleImage from '../components/ArticleImage'

it('renders correctly', () => {
  const subject = mount(<ArticleImage />)
  expect(EnzymeToJson(subject)).toMatchSnapshot()
})
