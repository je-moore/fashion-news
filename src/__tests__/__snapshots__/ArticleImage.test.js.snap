// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<ArticleImage>
  <img
    alt=""
    className="makeStyles-articleImg-1 makeStyles-defaultImg-2"
    src="https://fashionunited.info/global-assets/img/default/fu-default_1200x630_black-favicon.jpg"
  />
</ArticleImage>
`;
